### ISSUE TITLE
Working on a MSI B450M Bazooka (sort of)

### ISSUE THREAD

**Issue by ElSebita** [nagisa/msi-rgb#52]
Hello!

I tried `msi-rgb` with a MSI B450M Bazooka and it worked! I can set colors, blinks and effects as shown in the examples. I'm using Fedora 29 KDE.

But I can only modify a LED strip that flows from half the board to the bottom, along the left-bottom edge of it. The mobo has four leds on the back, always-on in white and not affected by `msi-rgb`. MSI [shows in its site](https://www.msi.com/Motherboard/B450M-BAZOOKA.html) that the official utility changes those LEDs. Perhaps another port to fiddle with them?

This is the output of `--verbose` command:

```
sudo ./msi-rgb --verbose ffffffff 88888888 22222222
Bank 12 (d0...100):
ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff
e0 00 00 00 00 00 00 00 ff ff 11 00 04 20 01 40
ff ff 00 00 ff ff 00 00 ff ff 00 00 00 00 19 e2
Bank 09 (20...40):
d3 52 ff 00 00 00 40 13 00 ff 40 01 11 00 00 11
0f ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff
Bank 0b (60...70):
0a 20 0a 40 00 00 ff ff ff ff ff ff ff ff ff ff
Chip identifier is: d352
```

Thank you four your utility!

**Comment by Geknip** [comment]
I've tried this on my own B450M Bazooka as well and it does work on that bottom left corner region but it also appropriately controls my Hyper 212 RGB cooler.

**Comment by ElSebita** [comment]
Indeed. Controlling both mobo and Coolermaster MasterLiquid ML240L RGB.
